app/views/shared/_messages.html.slim

```diff
@@ -1,6 +1,7 @@
 - unless flash.empty?
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
-      .mxn2.sm-mxn6.sm-mtn3
-        div class="px2 py1 sm-px6 mb-40p h5 line-height-2 alert alert-#{name}" role='alert'
-          .alert-inner = msg
+      div class="alert-#{name}" role='alert'
+        div class="mxn1 mtn1 alert-icon"
```

Comment:
with slim, this line can be simplified to:

`.mxn1.mtn1.alert-icon`

(the div is assumed)